client/redux/cart.js

```diff
@@ -0,0 +1,119 @@
+import axios from 'axios';
+import { ADD_CART_ITEM, SET_CART_ITEMS, ADD_MULTIPLE_CART_ITEMS, CHANGE_CART_ITEM, REMOVE_CART_ITEM } from './actionConstants';
+import { addLocalCartItem, changeLocalCartItem, removeLocalCartItem } from '../localCart/'
+
+// ACTION CREATORS
+export const _addCartItem = cartItem => {
+  return {
+    type: ADD_CART_ITEM,
+    cartItem
+  }
+};
+
+export const _changeCartItem = cartItem => {
+  return {
+    type: CHANGE_CART_ITEM,
+    cartItem
+  }
+};
+
+export const _removeCartItem = cartItem => {
+  return {
+    type: REMOVE_CART_ITEM,
+    cartItem
+  }
+};
+
+export const _setCartItems = cartItems => {
+  return {
+    type: SET_CART_ITEMS,
+    cartItems
+  }
+}
+
+export const _addMultipleCartItems = cartItems => {
+  return {
+    type: ADD_MULTIPLE_CART_ITEMS,
+    cartItems
+  }
+}
+
+// THUNK CREATORS
+export const addCartItem = (cartItem, isLoggedIn = false) => {
```

Comment:
since by default isLoggedIn is false, i think that means you don't have to pass in isLoggedIn if it isn't true in the cart component, lines 34 - 44.

Reply:
That's true, but when these thunk creators are called in the Cart component or SingleArt component, I don't know whether it's true or not.